### DISCUSSION TITLE
혹시 Utils를 제공해주실 수 있을까요...?

### DISCUSSION THREAD

**Discussion by channoori** [nsoojin/MiniSuperApp-fastcampus#2] [아무 질문이나 환영합니다] [answered]
강의를 시청중에 어디에도 찾을 수 없는 Utils파일들이 있어서 조금 불편하게 느껴집니다....ㅜㅜ
TableView+Utils
UIImage+Utils
이 3개가 강의 내용중에 빠져있어서 혼동이 되는 것 같습니다. 

Samples에 추가해 주시면 더 좋을 것 같습니다!
강의 잘 보고 있습니다 감사합니다!

**Comment by nsoojin** [comment, marked as answer]
아 죄송합니다. 빠진 파일추가했습니다. main 브랜치에서 다시 pull 받아보시기 바랍니다.
